src/js/actions/ContextIdActions.js

```diff
@@ -16,7 +16,7 @@ import {
   getGroupsData,
   getProjectSaveLocation
 } from "../selectors";
-import Repo from "../helpers/Repo";
+import Repo, {isSaving} from "../helpers/Repo";
```

Comment:
I'm not a huge fan of this method of checking if a save is occurring. If we're going to have that code in the Repo class let's abstract it away 100% so it's not confusing to people trying to use it later.
I originally didn't want to put that code in the Repo class because I wanted to keep things as flexible as possible.

However, how about we add a new `saveDebounced` method to `Repo` that's calling `save` but first checking to see if it's saving. Then don't expose the `isSaving` at all. 
This way we'll have the optional to use either `save` or `saveDebounced` as needed.

> NOTE: `saveDebounced` could return a boolean to indicate if the save was initiated.